### DISCUSSION TITLE
Creating picklist

### DISCUSSION THREAD

**Discussion by m4ax92** [YetiForceCompany/YetiForceCRM#14666] [How to] [answered]
Can I create a new custom picklist? i can't find the mentioned option anywhere

**Comment by rskrzypczak** [comment, marked as answer]
you can search in google :) https://www.google.com/search?tbm=vid&q=yetiforce+picklist
Go to the Edit Fields panel and add a new field.
![image](https://user-images.githubusercontent.com/10221999/118254344-4a117f00-b4ab-11eb-99f1-f96d80598bc3.png)

**Reply by m4ax92** [reply]
I didn't think about it... :( thanks!
